Purchases/Public/Purchases.swift

```diff
@@ -956,12 +957,12 @@ public extension Purchases {
      *
      * - Note: Subscription groups are automatically collected for determining eligibility. If RevenueCat can't
      * definitively compute the eligibilty, most likely because of missing group information, it will return
-     * `RCIntroEligibilityStatusUnknown`. The best course of action on unknown status is to display the non-intro
+     * `IntroEligibilityStatusUnknown`. The best course of action on unknown status is to display the non-intro
```

Comment:
`IntroEligibilityStatus.unknown`

Reply:
All the things like this should be fixed now.